lib/cluster/index.js

```diff
@@ -57,7 +57,7 @@ function Cluster(startupNodes, options) {
 
   var _this = this;
   this.connectionPool.on('-node', function (redis) {
-    if (_this.subscriber === redis) {
+    if (_this.status === 'ready' && _this.subscriber === redis) {
```

Comment:
We should export possible statuses to an enum to avoid having typos.
something like:

`_this.status = Redis.Statuses.READY`